genon/preprocessor/facade/gitbook_doc/code_serving_dev_manual.md

```
@@ -776,26 +776,46 @@ class DocumentProcessor:          # ← 클래스 이름 고정. main.py 가 이
 | `self._gr_cfg` | 개인정보 분류 설정 |
 | `self._page_desc_options` | PPT 페이지 설명 옵션 |
 
-#### `__call__` (2593–2704) — 확장자 라우팅
+#### `__call__` (2725–2875) — 확장자 라우팅
 
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)